translate/storage/test_xliff.py

```diff
@@ -321,7 +321,7 @@ def test_fuzzy(self):
         unit.markfuzzy()
         assert not unit.isfuzzy()  # untranslated
         unit.target = "Konsep"
-        assert unit.isfuzzy()
+        assert not unit.isfuzzy()  # translated
```

Comment:
This test is a bit confusing, but the new behavior looks more correct to me. Hopefully it won't break anything else.

Reply:
Yes, I think previous behavior was broken. It doesn't make that if you set a translation the unit is fuzzy automagically.